ext/fetch/26_fetch.js

```diff
@@ -591,4 +591,38 @@ function handleWasmStreaming(source, rid) {
   }
 }
 
-export { fetch, handleWasmStreaming };
+/**
+ * @param {string} url
+ * @param {BodyInit | null} data
+ */
+async function sendBeacon(url, data = null) {
+  const contentType = data === null ? null : extractBody(data).contentType;
+  try {
+    const ret = await mainFetch(
```

Comment:
**issue:** Spec states
> The user agent SHOULD schedule transmission of provided data to minimize resource (CPU and network) contention with other time-critical and high priority work.

We can't really say we're doing what the spec is trying to suggest here, although it is true that the spec doesn't say MUST here. We directly initiate the transmission of the provided data, we do not do any coalescing of beacon rqeuests, we do not do any prioritising to avoid competing with time-critical operations and higher priority network requests etc.